packages/mgt-components/src/components/sub-components/mgt-arrow-options/mgt-arrow-options.ts

```diff
@@ -104,7 +103,7 @@ export class MgtArrowOptions extends MgtBaseComponent {
    */
   public render() {
     return html`
-      <span class="Header" @click=${e => this.onHeaderClick(e)}>
+      <span class="Header" @click=${this.onHeaderClick}>
```

Comment:
🎉 Love seeing these little fixes as we go! getting rid of the unnecessary anonymous function that was being created on each render cycle is a really nice micro improvement!